source/config/profileUpgradeSteps.py

```diff
@@ -376,3 +376,19 @@ def upgradeConfigFrom_10_to_11(profile: ConfigObj) -> None:
 			"hidBrailleStandard added to braille display auto detection excluded displays. "
 			f"List is now: {profile['braille']['auto']['excludedDisplays']}"
 		)
+
+
+def upgradeConfigFrom_11_to_12(profile: ConfigObj) -> None:
+	"""
+	use Screen layout where supported is disabled by default.
+	If the user has the old default value <true>, preserve
+	the old value.
+	Otherwise, the new default value <false>, will be used.
+	"""
+	try:
+		useScreenLayout: str = profile['virtualBuffers']['useScreenLayout']
+		del profile['virtualBuffers']['useScreenLayout']
+	except KeyError:
+		useScreenLayout = True
+	if configobj.validate.is_boolean(useScreenLayout):
+		profile['virtualBuffers']['useScreenLayout'] = "true"

```

Comment:
please use double quotes consistently

```suggestion
		useScreenLayout: str = profile["virtualBuffers"]["useScreenLayout"]
		del profile["virtualBuffers"]["useScreenLayout"]
	except KeyError:
		useScreenLayout = True
	if configobj.validate.is_boolean(useScreenLayout):
		profile["virtualBuffers"]["useScreenLayout"] = "true"
```